ScanToShop/Managers/AlertManager.h

```diff
@@ -10,9 +10,16 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface AlertManager : UIViewController
+@interface AlertManager : NSObject
 
-+ (void)loginAlert:(NSString *)errorString ViewController:(UIViewController *) vc;
+typedef NS_ENUM(NSInteger, errorType) {
+    LoginErrorMissingInput,
+    ParseBackendError,
```

Comment:
This leaks parse implementation detail. Probably better to name "ServerError" or something.